### DISCUSSION TITLE
How do I get the hostname of the host running Neovim when using `--remote-tcp`?

### DISCUSSION THREAD

**Discussion by Hubro** [neovide/neovide#1269] [Q&A]
I have some `if` blocks in my nvim config that sets the `guifont` based on the current hostname. This is because my laptops both have high DPI screens, while my workstation does not.

Lately I'm experimenting with running nvim headless on my workstation and connecting with Neovide from a laptop. Unfortunately that means that the `guifont` is set on the workstation, meaning that my GUI font is absolutely tiny and unreadable on my laptops.

Is there any way to check the hostname of the host running the GUI? Is this something Neovide would have to provide through a global variable?

**Comment by MultisampledNight** [comment]
Mmmh, not directly. But you could get to your goal through other ways.

Assuming you're using an Unix-alike, if the host running Neovide is using X11 as display protocol, you could use the environment variable `WINIT_X11_SCALE_FACTOR` to control the scale Neovide uses. On Wayland, you can configure your compositor to do so.

See https://docs.rs/winit/0.26.1/winit/dpi/index.html#how-is-the-scale-factor-calculated.

**Reply by Hubro** [reply]
Unfortunately the laptop running the Neovide GUI is a Macbook :frowning_face:
